### PR TITLE
Prepare communication packages for May Patch Release

### PR DESCRIPTION
**Pull request by ankitarorabit**
All the packages are updated to ensure that there are no dependency conflicts
* Chat, Common -> 1.0.1
* PhoneNumbers, SMS -> 1.0.2
* Identity -> 1.1.0

### CODE REVIEW

File: sdk/communication/azure-communication-phonenumbers/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Release History
-## 1.1.0-beta.1 (Unreleased)
 
-## 1.0.1 (2021-04-30)
```
Comment (JianpingChen) [inline sdk/communication/azure-communication-phonenumbers/CHANGELOG.md:4]: keep the line for 1.0.1

**Comment by JianpingChen** [comment]
Why are we making an out-of-band release? June release is pretty soon

**Comment by JianpingChen** [comment]
Update, test coverage. eng/jacoco-test-coverage/pom.xml with new versions

**Comment by ankitarorabit** [comment]
> Update, test coverage. eng/jacoco-test-coverage/pom.xml with new versions

You mean eng/jacoco-test-coverage/pom.xml? it's already part of the PR

**Comment by ankitarorabit** [comment]
> Why are we making an out-of-band release? June release is pretty soon

Yes, it is soon but since we are already delayed with this recommended patch, will prefer sooner rather than later. We can skip the June release now.

File: sdk/communication/azure-communication-chat/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
+## 1.0.1 (2021-05-26)
+- Dependency versions updated.
```
Comment (g2vinay) [inline sdk/communication/azure-communication-chat/CHANGELOG.md:4]: Good to add the below header here and in other changelogs.

### Dependency Updates
- upgraded azure-core dependency to 1.16.0
- <Mention-other-dependencies that got updated>

**Review by JianpingChen — APPROVED** [review]

File: sdk/communication/azure-communication-chat/CHANGELOG.md (hunk 2 above)
Comment (ankitarorabit) [reply sdk/communication/azure-communication-chat/CHANGELOG.md:4]: I guess thats going to be a lot of information and hard to maintain the practice overtime,  so lets keep it simple

**Pull request merged**